webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc

```diff
@@ -0,0 +1,34 @@
+== *Create, Share and Consume Devfiles*
+:imagesdir: ../../../images
+'''''
+Organizations looking to standardize their development environment can do so by adopting devfiles.
+In the simplest case, developers can just consume the devfiles that are available from the public community registry.  If your organization needs custom devfiles that are authored and shared internally, then you will need a role based approach so developers, devfile authors, and registry administrators can interact together.
+
+image::devfileEcosystem.svg[]
+
+
+=== *The Main Areas of the Devfile Ecosystem*
+=== *Create*
+
+A devfile author, also known as a runtime provider can be an individual or a group representing a runtime vendor.  Devfile authors need sound knowledge of the supported runtime so they can create devfiles to build and run applications.
```

Comment:
Missing comma:

`A devfile, author, also known as a runtime provider, can be an individual...`

Reply:
updated.  Thanks again for the very thorough review @jc-berger !